### PR TITLE
Add documentation for deploy apim self host agent

### PR DESCRIPTION
**Pull request by duongthaiha**
This is the documentation for the [PR]#2316(https://github.com/microsoft/azure_arc/pull/2316) for the Issue APIM Self Host Extension #[1974](https://github.com/microsoft/azure_arc/issues/1974) please This to demonstrate ability to run workload on ARC with API still be centralise control on Azure as other Arc Service. 
Sorry I dont think I has the permission to link item the development please

**Summarization:**
This pull request adds a new section to the `docs/azure_jumpstart_arcbox/Full/_index.md` document. The new section provides instructions on deploying the Adventure Work WebAPI workload on a k3s cluster together with Azure API Management. It includes a PowerShell command to start the deployment, a list of steps that the script will perform, and screenshots showing the output of the deployment and a test request to the Adventure Workload.

Main documentation change:

* <a href="diffhunk://#diff-afa25513445734e6e1e0bc1335230bfaef50e390db0f6e0481ba72005e6873dbR580-R612">`docs/azure_jumpstart_arcbox/Full/_index.md`</a>: Added a new section providing instructions on deploying the Adventure Work WebAPI workload on a k3s cluster together with Azure API Management.

### CODE REVIEW

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 1]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
 - The below screenshot shows an email alert sent by Defender for Cloud when a SQL threat is detected. By default, this email is sent to the registered contact email at the subscription level.
   ![Screenshot showing test script results](./brute-force-attack-alert.png)
 
+### Adventure Workload and Azure API Management
+
+This section guide you through deploy Adventure Work WebAPI workload on k3s cluster together with Azure API Management to bring all the control panel on Azure inline with other ARC service.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:582]: Can you please check it is "Adventure Work" or "Adventure Works"? Also it is not ARC, it has to be Azure Arc. Standalone Arc is not recommended in our product documentation.

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 2]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
 - The below screenshot shows an email alert sent by Defender for Cloud when a SQL threat is detected. By default, this email is sent to the registered contact email at the subscription level.
   ![Screenshot showing test script results](./brute-force-attack-alert.png)
 
+### Adventure Workload and Azure API Management
+
+This section guide you through deploy Adventure Work WebAPI workload on k3s cluster together with Azure API Management to bring all the control panel on Azure inline with other ARC service. 
+
+> Note: The assumption is the Arc SQL Managed Instance has been deployed. The deployment start once the SQL Managed Instance has completed.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:584]:
```suggestion
> Note: The assumption is the Arc-enabled SQL Managed Instance has been deployed. The deployment starts once the SQL Managed Instance has been created.
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 3]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+
+- Start deployment by powershell command.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:586]:
```suggestion
- Start deployment by PowerShell command.
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 4]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+
+``` powershell
+  C:\ArcBox\Deploy-APIM.ps1
+```
+
+- The script will do the following:
+   - Deploy Adventure Web API to k3s.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:593]: Please keep the consistent name for Adventure Work or Adventure Works throughout the documentation.

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 5]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+   - Set back end of the Web API to AdventureWork SQL Managed Instance.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:594]:
```suggestion
   - Set the backend of the Web API to AdventureWork SQL Managed Instance.
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 6]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+   - Deploy Azure API Management with self host gateway.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:595]:
```suggestion
   - Deploy Azure API Management with the self-hosted gateway.
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 7]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+   - Deploy Self-host gate way to the k3s
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:596]:
```suggestion
   - Deploy self-hosted gateway to the k3s
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 8]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+   - Configure the connectively from Azure API Management, Self-host gateway and Adventure Workload
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:597]:
```suggestion
   - Configure the connectivity from Azure API Management, self-hosted gateway, and AdventureWorks API.
```

Comment (chintalavr) [reply docs/azure_jumpstart_arcbox/Full/_index.md:597]: Also please verify AdventureWorks or Adventure Works. I believe it is AdventureWorks.

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 9]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+
+- Deployment will finish show the following message
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:599]: Consider rephrasing this sentence.

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 10]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+  ![Screenshot showing Terminal screenshot show output of the deployment ](./apim_01_deploymentcomplete.png)
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:600]:
```suggestion
  ![Screenshot showing terminal output of the deployment ](./apim_01_deploymentcomplete.png)
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 11]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+
+ - Get the IP address for the selft host agent
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:602]:
```suggestion
 - Get the IP address for the self-hosted gateway
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 12]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+ 
+ ``` powershell
+ C:\ArcBox\kubectl get svc
+ ```
+ - The selfhost agent IP as below
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:607]:
```suggestion
 - The self-hosted gateway IP as below
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 13]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+ ![Screenshot showing Terminal screenshot show IP of the self host agent service ](./apim_02_selfhost_ip.png)
+
+ - Adventure Workload can be tested using the IP
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:610]: Please see the above comments regarding the naming consistency.

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 14]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+![Screenshot showing Terminal screenshot show output of the deployment ](./apim_10_request.png)
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:611]:
```suggestion
![Screenshot showing terminal output of the deployment ](./apim_10_request.png)
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 15]
```diff
@@ -577,6 +577,39 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+
 ### Included tools
 
 The following tools are including on the _ArcBox-Client_ VM.
```
Comment (chintalavr) [inline docs/azure_jumpstart_arcbox/Full/_index.md:615]:
```suggestion
The following tools are included in the _ArcBox-Client_ VM.
```

**Review by chintalavr — CHANGES_REQUESTED** [review]
Please go through the review comments and update the document as needed.

File: docs/azure_jumpstart_arcbox/Full/_index.md (hunk 9 above)
Comment (duongthaiha) [reply docs/azure_jumpstart_arcbox/Full/_index.md:599]: I have rephase it a little bit to be align with the rest. Please let me know if it is ok now. Any feedback or suggestion is much appreciated.

**Review by duongthaiha — COMMENTED** [review]
Include all the suggestions
Align all the workload name to AdventureWorks
Rephrase the wording to align with the others.

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 16]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+```
+
+The following tasks will be performed by the deployment:
+   - Deploy AdventureWorks API to k3s.
+   - Set the backend of the AdventureWorks API to AdventureWorks SQL Managed Instance.
+   - Deploy Azure API Management with the self-hosted gateway.
+   - Deploy self-hosted gateway to the k3s
+   - Configure the connectivity from Azure API Management, self-hosted gateway, and AdventureWorks API.
+
+- Deployment will finish show the following message
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/Full/_index.md:599]:
```suggestion
- Once deployment is complete you should see the following message:
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 17]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+  ![Screenshot showing terminal output of the deployment ](./apim_01_deploymentcomplete.png)
+
+ - Get the IP address for the self-hosted gateway
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/Full/_index.md:602]:
```suggestion
 - Get the IP address for the self-hosted gateway:
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 18]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
 - The below screenshot shows an email alert sent by Defender for Cloud when a SQL threat is detected. By default, this email is sent to the registered contact email at the subscription level.
   ![Screenshot showing test script results](./brute-force-attack-alert.png)
 
+### AdventureWorks API and Azure API Management
+
+This section guide you through deploy AdventureWorks WebAPI workload on k3s cluster together with Azure API Management to bring all the control panel on Azure inline with other Azure Arc service.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/Full/_index.md:582]:
```suggestion
This section will guide you through deploying an AdventureWorks WebAPI workload on the ArcBox-K3s cluster together with Azure API Management. 
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 19]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
 - The below screenshot shows an email alert sent by Defender for Cloud when a SQL threat is detected. By default, this email is sent to the registered contact email at the subscription level.
   ![Screenshot showing test script results](./brute-force-attack-alert.png)
 
+### AdventureWorks API and Azure API Management
+
+This section guide you through deploy AdventureWorks WebAPI workload on k3s cluster together with Azure API Management to bring all the control panel on Azure inline with other Azure Arc service. 
+
+> Note: The assumption is the Arc-enabled SQL Managed Instance has been deployed. The deployment starts once the SQL Managed Instance has been created.
+
+- Start deployment by PowerShell command.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/Full/_index.md:586]:
```suggestion
- Start the deployment by running this PowerShell command:
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 20]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+
+- Deployment will finish show the following message
+  ![Screenshot showing terminal output of the deployment ](./apim_01_deploymentcomplete.png)
+
+ - Get the IP address for the self-hosted gateway
+ 
+ ``` powershell
+ C:\ArcBox\kubectl get svc
+ ```
+ - The self-hosted gateway IP as below
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/Full/_index.md:607]:
```suggestion
 - The self-hosted gateway IP should look like the following image:
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 21]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+ ![Screenshot showing Terminal screenshot show IP of the self host agent service ](./apim_02_selfhost_ip.png)
+
+ - AdventureWorks API can be tested using the IP
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/Full/_index.md:610]:
```suggestion
 - The AdventureWorks API can be tested using the IP:
```

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
Made some suggested changes for grammar. Please review and incorporate.

**Review by duongthaiha — COMMENTED** [review]
Include suggestions from the PR to make the instruction better

**Review by dkirby-ms — APPROVED** [review]

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 22]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
 - The below screenshot shows an email alert sent by Defender for Cloud when a SQL threat is detected. By default, this email is sent to the registered contact email at the subscription level.
   ![Screenshot showing test script results](./brute-force-attack-alert.png)
 
+### AdventureWorks API and Azure API Management
+
+This section guide you through deploy AdventureWorks WebAPI workload on k3s cluster together with Azure API Management to bring all the control panel on Azure inline with other Azure Arc service. 
+
+> Note: The assumption is the Arc-enabled SQL Managed Instance has been deployed. The deployment starts once the SQL Managed Instance has been created.
```
Comment (likamrat) [inline docs/azure_jumpstart_arcbox/Full/_index.md:584]:
```suggestion
> **Note:** The assumption is the Arc-enabled SQL Managed Instance has been deployed. The deployment starts once the SQL Managed Instance has been created.
```

File: docs/azure_jumpstart_arcbox/Full/_index.md [hunk 23]
```diff
@@ -577,9 +577,42 @@ Please note it may take some time to show this status in the Azure portal, but s
...
+ - The self-hosted gateway IP as below
+ ![Screenshot showing Terminal screenshot show IP of the self host agent service ](./apim_02_selfhost_ip.png)
+
+ - AdventureWorks API can be tested using the IP
+![Screenshot showing terminal output of the deployment ](./apim_10_request.png)
+
 ### Included tools
 
-The following tools are including on the _ArcBox-Client_ VM.
+The following tools are included in the _ArcBox-Client_ VM.
```
Comment (likamrat) [inline docs/azure_jumpstart_arcbox/Full/_index.md:615]:
```suggestion
The following tools are included in the *ArcBox-Client* VM.
```

Comment (likamrat) [reply docs/azure_jumpstart_arcbox/Full/_index.md:615]: Please change italic text md syntax from underscore to * in the entire doc.

**Review by likamrat — APPROVED** [review]
Made a minor commit. LGTM and ready to merge. 

Great work @duongthaiha !

File: docs/azure_jumpstart_arcbox/Full/_index.md (hunk 4 above)
Comment (duongthaiha) [reply docs/azure_jumpstart_arcbox/Full/_index.md]: YesI have update so it should be AdventureWorks

**Review by chintalavr — APPROVED** [review]
Looks good with the suggested changes.

**Pull request merged**